try {
    hello();
} catch (error){
    console.log(error);
}

try{
    anotherFn();
} catch{
    console.log('Etos es un error')
}


    let a = 2;

    function b(){
        console.log(a);
    }

    b()